label_sync/cluster/label_sync_cron_job.yaml

```diff
@@ -15,15 +15,36 @@
 apiVersion: batch/v2alpha1
 kind: CronJob
 metadata:
-  name: label_sync_cron_job
+  name: label-sync-cron-job
 spec:
   schedule: "17 23 * * 0"    # Every Sunday 23:17 / 11:17PM
   jobTemplate:
     spec:
       template:
         spec:
           containers:
-            - name: label_sync
-              image: gcr.io/k8s-prow/label_sync:0.100
-          restartPolicy: OnFailure
+            - name: label-sync
+              image: gcr.io/k8s-testimages/label_sync:v20171121-90506fba
+              args:
+              - --config=/etc/config/labels.yaml
+              - --dry-run=false
+              - --orgs=kubernetes
+              - --token=/etc/github/oauth
+              imagePullPolicy: Always
+              volumeMounts:
+              - name: oauth
+                mountPath: /etc/github
+                readOnly: true
+              - name: config
+                mountPath: /etc/config
+                readOnly: true
+          restartPolicy: Never
```

Comment:
why is restartPolicy set to never?

Reply:
So that problems eventually result in the thing stopping, rather than crash looping and consuming all the tokens for prow.

Reply:
SGTM

Reply:
Ack